src/mlpack/methods/ann/init_rules/he_init.hpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file he_init.hpp
+ * @author kris singh
+ *
+ * Intialization rule for the neural networks. This simple initialization is
+ * performed by assigning a weights from a uniform distribution between 2/fan_in + fan_out.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_INIT_RULES_HE_INIT_HPP
+#define MLPACK_METHODS_ANN_INIT_RULES_HE_INIT_HPP
+
+#include <mlpack/prereqs.hpp>
+
+#include "he_uniform.hpp"
+#include "he_normal.hpp"
+ 
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * Base class for He Intilisation Policy
+ * Refrence 
+ * Kaiming He et al. (2015):
+ * Delving deep into rectifiers: Surpassing human-level performance on
+ * imagenet classification. arXiv preprint arXiv:1502.01852.
+ */
+template<typename InitializerType>
+class HeInit: public InitializerType
```

Comment:
Not sure why we have to inherent from the InitializerType here. Couldn't we just do 

```
InitializerType init;
init. Initialize(...);
````

Maybe I missed something?